test_rbd_thick_provisioning failing due to timeout issue

Test case ```tests/manage/pv_services/test_rbd_thick_provisioning.py::TestRbdThickProvisioning::test_rbd_thick_provisioning``` is failing with the below error.

```
tests/manage/pv_services/test_rbd_thick_provisioning.py:61: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/conftest.py:1540: in factory
    helpers.wait_for_resource_state(pvc_obj, status, timeout=timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

```
```
>           raise ResourceWrongStatusException(resource.name, resource.describe())
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource pvc-test-82048dec3e0842529bc0c2e4471e8a2 describe output: Name:          pvc-test-82048dec3e0842529bc0c2e4471e8a2
E           Namespace:     namespace-test-a181638cccf84c0e99f9934b1
E           StorageClass:  storageclass-test-rbd-2a4c43d22499402ca5
E           Status:        Bound
E           Volume:        pvc-92e4057a-ae0f-434a-a62b-559b534e810b
E           Labels:        <none>
E           Annotations:   pv.kubernetes.io/bind-completed: yes
E                          pv.kubernetes.io/bound-by-controller: yes
E                          volume.beta.kubernetes.io/storage-provisioner: openshift-storage.rbd.csi.ceph.com
E           Finalizers:    [kubernetes.io/pvc-protection]
E           Capacity:      5Gi
E           Access Modes:  RWO
E           VolumeMode:    Filesystem
E           Used By:       <none>
E           Events:
E             Type    Reason                 Age                 From                                                                                                                Message
E             ----    ------                 ----                ----                                                                                                                -------
E             Normal  Provisioning           2m1s                openshift-storage.rbd.csi.ceph.com_csi-rbdplugin-provisioner-85958b7b6d-t8w5w_95ca962d-81a8-45ef-bbe0-a28215540ae3  External provisioner is provisioning volume for claim "namespace-test-a181638cccf84c0e99f9934b1/pvc-test-82048dec3e0842529bc0c2e4471e8a2"
E             Normal  ExternalProvisioning   15s (x9 over 2m1s)  persistentvolume-controller                                                                                         waiting for a volume to be created, either by external provisioner "openshift-storage.rbd.csi.ceph.com" or manually created by system administrator
E             Normal  ProvisioningSucceeded  1s                  openshift-storage.rbd.csi.ceph.com_csi-rbdplugin-provisioner-85958b7b6d-t8w5w_95ca962d-81a8-45ef-bbe0-a28215540ae3  Successfully provisioned volume pvc-92e4057a-ae0f-434a-a62b-559b534e810b

ocs_ci/helpers/helpers.py:119: ResourceWrongStatusException
```

The error is due to timeout waiting for the PVC to reach Bound state. timeout value was provided as 120 seconds. The PVC reached Bound state within 121 seconds.

--- a/tests/manage/pv_services/test_rbd_thick_provisioning.py
+++ b/tests/manage/pv_services/test_rbd_thick_provisioning.py
@@ -65,7 +65,7 @@ class TestRbdThickProvisioning(ManageTest):
                 access_modes=access_modes_rbd,
                 status=constants.STATUS_BOUND,
                 num_of_pvc=3,
-                timeout=120,
+                timeout=150,
             )
             for pvc_obj in pvc_objs:
                 pvc_obj.io_file_size = pvc_and_file_sizes[pvc_size]

